docs/status.md

```diff
@@ -144,5 +144,5 @@ one of the following tracking labels.
 | unary_einsum             | no            | revisit      | no             | no              | no          |
 | uniform_dequantize       | no            | yes*         | yes*           | yes             | no          |
 | uniform_quantize         | no            | yes*         | infeasible     | yes             | no          |
-| while                    | no            | yes*         | yes*           | revisit         | no          |
+| while                    | yes           | revisit      | yes            | revisit         | no          |
```

Comment:
Approved otherwise!

Do let me know why the verification is a `revisit` for while/If/Case?

Reply:
Checkout #365 
revisit is to finalise behaviour on 0 operands